rules/correct-tag-name.js

```diff
@@ -0,0 +1,34 @@
+const { dim, bold } = require('colors');
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'correct-tag-name';
```

Comment:
Suggest to rename. Or extend this logic to satisfy the rule meaning: tag name is correct if:
- tag is not a div
- tag is one of standard html tags
- tag is a custom element (so it consist dash inside name and the tag name and component name are the same)

Reply:
I extended logic.
I wrote rules to check if tag name is a valid html tag, if there are forbidden symbols in tag name